package.json

```
@@ -3,7 +3,7 @@
   "version": "2.5.1",
   "description": "LoopBack Connector common code for IBM databases",
   "engines": {
-    "node": ">=6"
+    "node": ">=9"
```

Please open a new pull request for dropping support for old Node.js version. That way people watching this repository have a chance to comment, and the CHANGELOG (release notes) gets a standalone entry.

Also note that odd-numbered major versions of Node.js are not eligible for long term support, the version 9 haven't been supported for a while. You can learn more here: https://github.com/nodejs/release

```suggestion
    "node": ">=10"
```

---

The PR https://github.com/strongloop/loopback-ibmdb/pull/81 set it to `>=8.9`.